python-prediction-algos/glm-binary-classification/algo.json

```diff
@@ -0,0 +1,140 @@
+/* This file is the descriptor for the Custom Python Prediction algorithm dev-glm_glm */
```

Comment:
8f0da96b6cb99cf988ca729e3bde1b5f316c1e38

Reply:
Hi @matthewgalloway,
Perhaps I missed something, but I can't see the renaming of the folder in  8f0da96
Could you double check?
Thanks,
Alex

Reply:
Apologies i had the wrong commit 70a78e96aa574501a305f06ca546df830585b22f

Reply:
Actually, the format is still wrong, the correct one should be "generalized-linear-models_binary-classification" instead of "generalized_linear_models-binary-classification".
Same for the regression

Reply:
Still waiting on this one. If we don't do this, our analytics won't work on this plugin.

Reply:
[02b5700](https://github.com/dataiku/dss-plugin-generalized-linear-models/commit/02b570066c59b09eb036f5082f57e8fbc3c981f6)